packages/react-events/src/Press.js

```diff
@@ -863,6 +871,30 @@ const PressResponder = {
             isKeyboardEvent = true;
           }
 
+          // Determine whether to call preventDefault on subsequent native events.
+          if (
+            context.isTargetWithinEventComponent(target) &&
+            context.isTargetWithinHostComponent(target, 'a', true)
+          ) {
+            const {
+              altKey,
+              ctrlKey,
+              metaKey,
+              shiftKey,
+            } = (nativeEvent: MouseEvent);
+            // Check "open in new window/tab" and "open context menu" key modifiers
+            const preventDefault = props.preventDefault;
+            if (
+              preventDefault !== false &&
+              !shiftKey &&
+              !metaKey &&
+              !ctrlKey &&
+              !altKey
+            ) {
+              state.shouldPreventDefault = true;
```

Comment:
This state needs to be reset somewhere

Reply:
It should be reset in `pointerup` right?

Reply:
This is within the `pointerup` event processing. It looks like this code is only for handling `click`, so I can make it explicitly for that and reset it in the `click` block.

Reply:
No I mean, we should reset `shouldPreventDefault` to `false` in `pointerup` if we don't set it to `true`. That way we don't wrongly prevent things we shouldn't.